apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -734,9 +814,7 @@ const UniversitySearch: Screen<UniversitySearchProps> = ({
                       variant="text"
                       icon="reload"
                       size="small"
-                      onClick={() =>
-                        setFilters(JSON.parse(JSON.stringify(initialFilters)))
-                      }
+                      onClick={() => clearFilterParams()}
```

Comment:
Probably enough to just do

onClick={clearFilterParams}

Reply:
Fixed